README.md

```diff
@@ -33,7 +33,3 @@ has the time to spare. If you want to work on the bridge, please see
 Please come visit us in the 
 [support room](https://matrix.to/#/#matrix_appservice_slack:cadair.com/) if you
 have any questions.
-
-# Documentation
-
-This documentation is hosted on [Read the Docs](https://matrix-appservice-slack.rtfd.io/).
```

Comment:
I know we have a badge for it, but it doesn't seem like a bad idea to have a second link to the documentation.

Reply:
There's also one in "Setting up". This was the third link to the same docs.